samples/client-sdk/knocking/index.js

```diff
@@ -112,6 +157,9 @@ const checkAccessLevel = async () => {
 };
 
 const handleJoinedMeeting = (e) => {
+  hideForms();
```

Comment:
(I'm going to commit a suggestion. Feel free to revert, though.)

Reply:
Fair enough! We can do that - I think last time if I accidentally clicked Join again, I'd get an error. Maybe we can _disable_ the join link when in the call without hiding the form?

(Also I just realized, we should probably have the token hidden and not pasted in plain-text in the token form. Basically treating it like a password field)

Reply:
I committed another option but feel free to remove it! I'm not tooooo worried about edge cases since this is a mostly-happy path example, but again, up to you! 
My thinking here is to maintain the 2x2 grid of forms and video containers to prevent shifting, but hiding the form that can't be used makes sense to me.

Reply:
(I'm going to not commit anything else because it feels weird on someone else's PR, it just seemed faster in this case since there were a few changes!)

Reply:
No worries at all, feel free to commit stuff :D I'll take a look here and run locally either tonight or tomorrow! (and will fix the "rejected from call" message visibility issue you caught below)

Reply:
Ty! In that case, I made the token input a password type in a new commit. The first commit I added should resolve the rejected issue but test to make sure 🙇

Reply:
Ok I just submitted one fix for duplicate DOM IDs, which triggered console warnings. LMK if anything else stands out!